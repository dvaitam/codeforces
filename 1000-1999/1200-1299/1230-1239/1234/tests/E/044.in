5 10
2 3 3 2 5 3 5 1 1 4
