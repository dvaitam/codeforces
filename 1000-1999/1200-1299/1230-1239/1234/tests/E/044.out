16 17 19 10 14
